### PR TITLE
Update HuaweiAds email

### PR DESCRIPTION
**Pull request by ahmetfaruk59**
Fixing https://github.com/prebid/prebid-server/issues/2626

### CODE REVIEW

**Comment by ahmetfaruk59** [comment]
Hi @bretg @gargcreation1992 
I changed the group email adress of HuaweiAds.

File: static/bidder-info/huaweiads.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 endpoint: "https://acd.op.hicloud.com/ppsadx/getResult"
 disabled: true
 maintainer:
-  email: hwads@huawei.com
+  email: prebid@huawei.com
```
Comment (Sonali-More-Xandr) [inline static/bidder-info/huaweiads.yaml:4]: @ahmetfaruk59 We have sent a verification email to `prebid@huawei.com`. Please reply to the verification mail with "received" message to complete the verification process. 
Thank you!

Comment (ahmetfaruk59) [reply static/bidder-info/huaweiads.yaml:4]: received

Comment (Sonali-More-Xandr) [reply static/bidder-info/huaweiads.yaml:4]: Verification completed!

**Review by Sonali-More-Xandr — APPROVED** [review]

File: static/bidder-info/huaweiads.yaml (hunk 1 above)
Comment (ahmetfaruk59) [reply static/bidder-info/huaweiads.yaml:4]: Thanks a lot!

**Review by gargcreation1992 — APPROVED** [review]

**Pull request merged**